model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -82,6 +84,25 @@ object AttributeName {
   }
 }
 
+object WorkspaceVersions {
+  sealed trait WorkspaceVersion extends ValueObject
+
+  case object V1 extends WorkspaceVersion {
+    override val value: String = "v1"
+  }
+  case object V2 extends WorkspaceVersion {
+    override val value: String = "v2"
+  }
+
+  def fromString(versionString: String): Option[WorkspaceVersion] = {
+    versionString match {
+      case V1.value => Option(V1)
+      case V2.value => Option(V2)
+      case _ => None
```

Comment:
Since this isn't nullable, should this error? (I don't know what code conventions are there yet)

Reply:
this is just the string conversion code... it is up to the caller to determine if this is an error